### PR TITLE
WolframAlpha Search

### PR DESCRIPTION
**Pull request by pierluigi-failla**
This will allow AutoGPT to interact with WolframAlpha. API Key (Wolfram AppID) is needed.

### CODE REVIEW

File: src/autogpt_plugins/wolframalpha_search/__init__.py [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+        Returns:
+            PromptGenerator: The prompt generator.
+        """
+
+        prompt.add_command(
+            "wolframalpha_search",
+            self._description,
+            {"query": "<query>"},
+            _wolframalpha_search,
+        )
```
Comment (ntindle) [inline src/autogpt_plugins/wolframalpha_search/__init__.py:224]: Check the api is stood up correctly

**Review by ntindle — CHANGES_REQUESTED** [review]
See comments

File: src/autogpt_plugins/wolframalpha_search/__init__.py (hunk 1 above)
Comment (pierluigi-failla) [reply src/autogpt_plugins/wolframalpha_search/__init__.py]: Correct! Good catch! I just released the fix!

File: src/autogpt_plugins/wikipedia_search/wikipedia_search.py [hunk 2]
```diff
@@ -54,6 +54,6 @@ def _wikipedia_search(query: str, num_results: int = 5) -> str | list[str]:
                 if len(items) == num_results:
                     break
         except Exception as e:
-            return f"'wikipedia_search' on query: {query} raised exception: {e}"
+            return f"'_wikipedia_search' on query: '{query}' raised exception: '{e}'"
```
Comment (ntindle) [inline src/autogpt_plugins/wikipedia_search/wikipedia_search.py:57]: remove this change, and fix in a separate PR

**Review by ntindle — CHANGES_REQUESTED** [review]
Add some tests

**Comment by ntindle** [comment]
Can you split this into two prs? one for the organization and another for the WA search?

**Comment by pierluigi-failla** [comment]
> 

yep, I reverted the Wiki change in this pr, I'm struggling with circ deps in the test... I followed the structure of Twitter example, but it seems not the best one for tests. I'm working on it.

**Review by ntindle — APPROVED** [review]

**Comment by ntindle** [comment]
Resolve conflicts

**Review by ntindle — APPROVED** [review]

**Pull request merged**